plutus-benchmark/common/PlutusBenchmark/Common.hs

```diff
@@ -11,11 +11,13 @@ module PlutusBenchmark.Common
     , toNamedDeBruijnTerm
     , compiledCodeToTerm
     , haskellValueToTerm
-    , benchTermCek
     , benchProgramCek
     , unsafeRunTermCek
     , runTermCek
     , cekResultMatchesHaskellValue
+    , mkEvalCtx
+    , evaluateCekLikeInProd
```

Comment:
It doesn't, but I want to have and export it, so that we can eventually have tests ensuring that the result of each program used in benchmarks doesn't change and for that it's worth using the same machinery that runs the benchmarks and that the ledger uses.